Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobRemoteProcess.java

```diff
@@ -259,6 +265,10 @@ public String getTitle() {
     Observable<SimpleImmutableEntry<String, String>> awaitForJobDone(ISparkBatchJob runningJob) {
         return runningJob.awaitDone()
                 .subscribeOn(schedulers.processBarVisibleAsync("Spark batch job " + getTitle() + " is running"))
+                .doOnNext(jobStateDiagnosticsPair ->
+                        getCtrlSubject().onNext(new SimpleImmutableEntry(
+                                MessageInfoType.Telemetry,
+                                jobStateDiagnosticsPair.getKey() + "`````" + jobStateDiagnosticsPair.getValue())))
```

Comment:
It might be more generic to apply JSON format for `jobStateDiagnosticsPair` serialized.